tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -100,20 +100,21 @@ def _test_job_dependencies(remote_command_executor, region, stack_name, scaledow
 
     jobs_execution_time = 1
     estimated_scaleup_time = 5
-    estimated_scaledown_time = 20
     asg_capacity_time_series, compute_nodes_time_series, timestamps = get_compute_nodes_allocation(
         scheduler_commands=slurm_commands,
         region=region,
         stack_name=stack_name,
         max_monitoring_time=minutes(jobs_execution_time)
         + minutes(scaledown_idletime)
         + minutes(estimated_scaleup_time)
-        + minutes(estimated_scaledown_time),
+        + minutes(10),
```

Comment:
I'd keep the `estimated_scaledown_time` variable because it is clearer than the number `10` added directly.

Reply:
done! called it max_scaledown_time